test/Microsoft.Kusto.ServiceLayer.UnitTests/FileBrowser/FileBrowserTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using Microsoft.Kusto.ServiceLayer.FileBrowser;
+using Microsoft.Kusto.ServiceLayer.FileBrowser.Contracts;
+using Xunit;
+
+namespace Microsoft.Kusto.ServiceLayer.UnitTests.FileBrowser
+{
+    /// <summary>
+    /// File browser unit tests
+    /// </summary>
+    public class FileBrowserTests
+    {
+        [Fact]
+        public void CreateFileBrowserOperationTest()
```

Comment:
So the test looks fine for what it is - but I guess the first question I have is do you actually need this filebrowser code for your stuff? And is it any different than the normal ServiceLayer? Because if you do and it isn't then I would rather see you make it so that the code is shared between the projects instead of duplicating it.